src/OVAL/probes/probe/main.c

```diff
@@ -189,14 +191,17 @@ int main(int argc, char *argv[])
 	oscap_set_verbose(verbosity_level, verbose_log_file, true);
 
 	if ((errno = pthread_barrier_init(&OSCAP_GSYM(th_barrier), NULL,
```

Comment:
Why not to use const ints to name the argument variables and discard those comments?
Then, this call to `pthread_barrier_init` that seems to have different number of arguments across platforms could be wrapped in an inline function. And you could use the preprocessor to pick between the two definitions and leave the call itself be without preprocessor.

Reply:
OK